Integration tests for Vulnerability Detector - Failed

## Description

Analyzing the [integration tests](https://github.com/wazuh/wazuh/issues/25347) executed with the new GHA, the following error was detected:

Test: https://github.com/wazuh/wazuh/actions/runs/10582087903

Error:

```
ImportError while loading conftest '/home/runner/work/wazuh/wazuh/tests/integration/test_vulnerability_detector/test_cpe_helper/conftest.py'.
test_vulnerability_detector/test_cpe_helper/__init__.py:5: in <module>
    from wazuh_testing.modules import vulnerability_detector as vd
E   ImportError: cannot import name 'vulnerability_detector' from 'wazuh_testing.modules' (/usr/local/lib/python3.10/dist-packages/wazuh_testing/modules/__init__.py)
Error: Process completed with exit code 4.
```


More research is required

--- a/tests/integration/test_vulnerability_detector/test_cpe_helper/__init__.py
+++ b/tests/integration/test_vulnerability_detector/test_cpe_helper/__init__.py
@@ -2,7 +2,7 @@
 # Created by Wazuh, Inc. <info@wazuh.com>.
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 from pathlib import Path
-from wazuh_testing.modules import vulnerability_detector as vd
+from wazuh_testing.modules.modulesd import vulnerability_detector as vd
 from wazuh_testing import FEEDS_PATH
 
 # Constants & base paths

--- a/tests/integration/test_vulnerability_detector/test_cpe_helper/test_cpe_indexing_missing_field.py
+++ b/tests/integration/test_vulnerability_detector/test_cpe_helper/test_cpe_indexing_missing_field.py
@@ -54,7 +54,7 @@ from wazuh_testing.utils.configuration import (load_configuration_template, get_
                                                update_configuration_template)
 from wazuh_testing.utils.callbacks import generate_callback
 from wazuh_testing.modules.modulesd.vulnerability_detector import patterns as cb
-from wazuh_testing.modules import vulnerability_detector as vd
+from wazuh_testing.modules.modulesd import vulnerability_detector as vd
 from wazuh_testing.modules.modulesd.configuration import MODULESD_DEBUG
 from wazuh_testing.modules.monitord.configuration import MONITORD_ROTATE_LOG
 from test_vulnerability_detector.utils import check_baseline_scan_start

--- a/tests/integration/test_vulnerability_detector/test_cpe_helper/test_cpe_indexing_wrong_tags.py
+++ b/tests/integration/test_vulnerability_detector/test_cpe_helper/test_cpe_indexing_wrong_tags.py
@@ -55,7 +55,7 @@ from wazuh_testing.utils.configuration import (load_configuration_template, get_
                                                update_configuration_template)
 from wazuh_testing.utils.callbacks import generate_callback
 from wazuh_testing.modules.modulesd.vulnerability_detector import patterns as cb
-from wazuh_testing.modules import vulnerability_detector as vd
+from wazuh_testing.modules.modulesd import vulnerability_detector as vd
 from wazuh_testing.modules.modulesd.configuration import MODULESD_DEBUG
 from wazuh_testing.modules.monitord.configuration import MONITORD_ROTATE_LOG
 from wazuh_testing.utils.mocking import VULNERABLE_PACKAGES

--- a/tests/integration/test_vulnerability_detector/test_cpe_helper/test_cpe_indexing_wrong_values.py
+++ b/tests/integration/test_vulnerability_detector/test_cpe_helper/test_cpe_indexing_wrong_values.py
@@ -55,7 +55,7 @@ from wazuh_testing.utils.configuration import (load_configuration_template, get_
                                                update_configuration_template)
 from wazuh_testing.utils.callbacks import generate_callback
 from wazuh_testing.modules.modulesd.vulnerability_detector import patterns as cb
-from wazuh_testing.modules import vulnerability_detector as vd
+from wazuh_testing.modules.modulesd import vulnerability_detector as vd
 from wazuh_testing.modules.modulesd.configuration import MODULESD_DEBUG
 from wazuh_testing.modules.monitord.configuration import MONITORD_ROTATE_LOG
 from wazuh_testing.utils.mocking import VULNERABLE_PACKAGES

--- a/tests/integration/test_vulnerability_detector/test_feeds/test_duplicate_feeds.py
+++ b/tests/integration/test_vulnerability_detector/test_feeds/test_duplicate_feeds.py
@@ -63,7 +63,7 @@ from wazuh_testing.tools.monitors.file_monitor import FileMonitor
 from wazuh_testing.utils.db_queries import cve_db
 from wazuh_testing.utils.configuration import (load_configuration_template, get_test_cases_data,
                                                update_configuration_template)
-from wazuh_testing.modules import vulnerability_detector as vd
+from wazuh_testing.modules.modulesd import vulnerability_detector as vd
 from wazuh_testing.modules.modulesd.configuration import MODULESD_DEBUG
 from wazuh_testing.modules.monitord.configuration import MONITORD_ROTATE_LOG
 from wazuh_testing.modules.modulesd.vulnerability_detector import patterns as cb

--- a/tests/integration/test_vulnerability_detector/test_general_settings/__init__.py
+++ b/tests/integration/test_vulnerability_detector/test_general_settings/__init__.py
@@ -3,7 +3,7 @@
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 from pathlib import Path
 from wazuh_testing import FEEDS_PATH
-from wazuh_testing.modules import vulnerability_detector as vd
+from wazuh_testing.modules.modulesd import vulnerability_detector as vd
 
 # Constants & base paths
 DATA_PATH = Path(Path(__file__).parent, 'data')

--- a/tests/integration/test_vulnerability_detector/test_providers/__init__.py
+++ b/tests/integration/test_vulnerability_detector/test_providers/__init__.py
@@ -3,7 +3,7 @@
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 from pathlib import Path
 from wazuh_testing import FEEDS_PATH
-from wazuh_testing.modules import vulnerability_detector as vd
+from wazuh_testing.modules.modulesd import vulnerability_detector as vd
 
 # Constants & base paths
 DATA_PATH = Path(Path(__file__).parent, 'data')

--- a/tests/integration/test_vulnerability_detector/test_scan_results/__init__.py
+++ b/tests/integration/test_vulnerability_detector/test_scan_results/__init__.py
@@ -3,7 +3,7 @@
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 from pathlib import Path
 from wazuh_testing import FEEDS_PATH
-from wazuh_testing.modules import vulnerability_detector as vd
+from wazuh_testing.modules.modulesd import vulnerability_detector as vd
 
 # Constants & base paths
 DATA_PATH = Path(Path(__file__).parent, 'data')

--- a/tests/integration/test_vulnerability_detector/test_scan_types/__init__.py
+++ b/tests/integration/test_vulnerability_detector/test_scan_types/__init__.py
@@ -3,7 +3,7 @@
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 from pathlib import Path
 from wazuh_testing import FEEDS_PATH
-from wazuh_testing.modules import vulnerability_detector as vd
+from wazuh_testing.modules.modulesd import vulnerability_detector as vd
 
 # Constants & base paths
 DATA_PATH = Path(Path(__file__).parent, 'data')

--- a/tests/integration/test_vulnerability_detector/test_vulnerability_inventory/__init__.py
+++ b/tests/integration/test_vulnerability_detector/test_vulnerability_inventory/__init__.py
@@ -3,7 +3,7 @@
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 from pathlib import Path
 from wazuh_testing import FEEDS_PATH
-from wazuh_testing.modules import vulnerability_detector as vd
+from wazuh_testing.modules.modulesd import vulnerability_detector as vd
 
 
 # Constants & base paths

